packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.js

```diff
@@ -155,7 +155,7 @@ export const ButtonsMixin = (superClass) =>
       if (container.offsetWidth < container.scrollWidth) {
         this._hasOverflow = true;
 
-        const isRTL = this.getAttribute('dir') === 'rtl';
+        const isRTL = this.__isRTL;
```

Comment:
Should we remove the `__isRTL` method also from `packages/menu-bar/src/vaadin-menu-bar-interactions-mixin.js`

Reply:
Couldn't find it there, seems like this mixin was the only place.